web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1290,7 +1318,7 @@ class AppVersionHistory extends Component {
                           {versionHistory.length > 1 && this.renderDiffBtn()}
                         </div>
                       </div>
-                      {this.renderAppVersionHistoryRow(versionHistory[0])}
```

Comment:
why not keep using `versionHistory[0]`? i don't get why it has to be a separate variable `latestAvailableVersion` in the state, will be harder to maintain/keep it updated